### PR TITLE
#149800557 Create add contribution icon dropdown

### PR DESCRIPTION
**Pull request by Andela-Didacus**
#### What does this PR do?
- add a drop down list for the add contribution icon. 

#### Description of Task to be completed?
Create a drop down list to display on hover on the add contribution icon on the navigation bar. The drop down should contain new idea, new skill and new resource.

#### How should this be manually tested?
After cloning the repository and installing the required dependencies using `yarn install`, run `npm test` to test.

#### Any background context you want to provide?
Initially the there was no drop down list on hovering over the add contribution icon, This has been made available by this PR.

#### What are the relevant pivotal tracker stories?
 
 [#149800557](https://www.pivotaltracker.com/story/show/149800557) 

#### Screenshots (if appropriate)
screen shot 2017-08-03 at 2 07 32 pm

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
@Andela-Didacus do you not see the excessive number of commits here?

File: src/common/navbar/NavBar.component.tsx [hunk 1]
```diff
@@ -38,12 +40,21 @@ export class NavBar extends React.Component <any, any> {
...
+                </div>
+                <div className="horizontal-line">
+                </div>
+                <div className="thrid-line">
+                </div>
               </div>
-              <div className="thrid-line">
+              <div className="drop-down-content contribution">
+                <div className="content-container">
+                  <Link to="/skills"><div className="content">New Idea</div></Link>
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.component.tsx]: ### Critical

Prefix these with `contribute`, i.e `/contribute/skills`

Comment (andela-kakpobome) [inline src/common/navbar/NavBar.component.tsx]: @BolajiOlajide @andela-aolasode does this makes sense?

File: src/common/navbar/NavBar.scss [hunk 2]
```diff
@@ -125,5 +125,63 @@
...
+        font-size: 1.1rem;
+        text-decoration: none;
+      }
+
+      .content-container {
+        height: 10.7rem;
+        border: solid 1px map-get($grey, 200);
+        border-radius: 3px;
+        padding-top: 0.7rem;
+        background-color: map-get($grey, 100);
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.scss]: ### Critical

Background should be on the top of this list.

File: src/common/navbar/NavBar.component.tsx (hunk 1 above)
Comment (andela-aolasode) [inline src/common/navbar/NavBar.component.tsx]: I think the link should be `/contribute/{contributionType}` e.g. `/contribute/idea`

**Pull request merged**